collectors/mesos/agent/agent.go

```diff
@@ -146,14 +146,19 @@ func getFrameworkInfoByFrameworkID(frameworkID string, frameworks []frameworkInf
 func getLabelsByContainerID(containerID string, frameworks []frameworkInfo, log *logrus.Entry) map[string]string {
 	labels := map[string]string{}
 	for _, framework := range frameworks {
-		log.Debugf("Attemping to add labels to %v framework", framework)
+		log.Debugf("Attempting to add labels to %v framework", framework)
 		for _, executor := range framework.Executors {
 			log.Debugf("Found executor %v for framework %v", framework, executor)
 			if executor.Container == containerID {
 				log.Debugf("ContainerID %v for executor %v is a match, adding labels", containerID, executor)
 				for _, pair := range executor.Labels {
-					log.Debugf("Adding label for containerID %v: %v = %+v", containerID, pair.Key, pair.Value)
-					labels[pair.Key] = pair.Value
+					if len(pair.Value) > 128 {
+						log.Warnf("Label %s is longer than 128 chars; discarding label", pair.Key)
+						log.Debugf("Discarded label value: %s", pair.Value)
```

Comment:
Instead of an if/else the usual way this would appear in the wild would be

```
if len(pair.Value) > 128 {
  log.warn(...)
  continue
}
labels[pair.Key] = pair.Value
```